lib/Controller/MentionController.php

```diff
@@ -39,14 +41,20 @@ public function mention(int $fileId, string $mention): DataResponse {
 			return new DataResponse([], Http::STATUS_NOT_FOUND);
 		}
 
-		$userFolder = $this->rootFolder->getUserFolder($mention);
+		$userResults = $this->userManager->searchDisplayName($mention, 1);
```

Comment:
I think this is a design flaw of the post message API that is not returning us a real unique identifier. Worth to file upstream to further discuss, but fine as a workaround with me now.

Reply:
File upstream where exactly? Should we rather have Collabora return the UID rather than display name?